include/RAJA/util/resource.hpp

```diff
@@ -21,7 +21,9 @@
 #define RAJA_resource_HPP
 
 #include "camp/resource.hpp"
+#if (defined(__NVCC__) || (defined(__clang__) && defined(__CUDA__))) && defined(RAJA_ENABLE_CUDA)
```

Comment:
The guards above enable us to have a single RAJA build for packages which use RAJA and are compiled with and without nvcc.